examples/load_finetune_huggingface_model/README.md

```diff
@@ -0,0 +1,42 @@
+# Multi-Agent Conversation with Custom Model Loading and Fine-Tuning in AgentScope
+
+This example demonstrates how to load and optionally fine-tune a Hugging Face model within a multi-agent conversation setup using AgentScope. The complete code is provided in `load_finetune_huggingface_model.py`.
+
+## Functionality Overview
+
+Compared to basic conversation setup, this example introduces model loading and fine-tuning features:
+
+- Use `dialog_agent.load_model(model_id, local_model_path)` to load a model either from the Hugging Face Model Hub or a local directory.
+- Apply `dialog_agent.fine_tune(data_path)` to fine-tune the model based on your dataset.
+
+The default hyperparameters for (SFT) fine-tuning are specified in `agentscope/src/agentscope/models/huggingface_model.py`. For customized hyperparameters, specify them in `model_configs` if the model needs to be fine-tuned at initialization, or specify through `fine_tune_config` in `Finetune_DialogAgent`'s `fine_tune` method after initialization, as shown in the example script `load_finetune_huggingface_model.py`.
```

Comment:
> specify them in `model_configs`
what is field in the `model_configs`?  `fine_tune_config`?

Reply:
Updated according to suggestion in `agentscope/examples/load_finetune_huggingface_model/load_finetune_huggingface_model.py`.